### ISSUE TITLE
Doesn't switch to unscaled mode

### ISSUE THREAD

**Issue by sarimarton** [jakehilborn/displayplacer#31]
I have two resolutions in use with Resolutionator:

1) 2560x1440 _unscaled_
2) 2560x1440 _scaled_

I want to easily switch between them on my 12" MacBook, because (1) significantly more resources from the mac, but is crispy on my 4K monitor, and (2) is a little blurry, but the mac is less under a load.

displayplacer identified the two modes as:
1) displayplacer "id:03B557AF-C729-02CE-0411-6ABB93A76C4D res:2560x1440 hz:60 color_depth:4 scaling:off origin:(0,0) degree:0"
2) displayplacer "id:03B557AF-C729-02CE-0411-6ABB93A76C4D res:2560x1440 hz:60 color_depth:8 scaling:on origin:(0,0) degree:0"

But both commands switches the monitor to scaled mode.

**Comment by jakehilborn** [comment]
Does displayplacer output any errors about not being able to find those parameters? Something like this? `Screen ID F466F621-B5FA-04A0-0800-CFA6C258DECD: could not find res:2560x1440 hz:60 color_depth:8 scaling:on`

**Comment by sarimarton** [comment]
Nope, no errors, no any output. It does its thing and returns to the command line. The monitor goes black for a second, and it comes back in scaled mode.

**Comment by jakehilborn** [comment]
Are you able to set the resolution to 2560x1440 using the system settings? I'm asking since I've noticed that macOS doesn't allow usage of some of the resolutions shown my displayplacer.

My suggestion is to execute `displayplacer list` and find the 2560x1440 modes. Try finding a mode that fits your requirements and use displayplacer to set that. For example, `displayplacer "id:03B557AF-C729-02CE-0411-6ABB93A76C4D res:2560x1440 mode:123 origin:(0,0) degree:0"`. It's possible that one of the 2560x1440 modes works while another one doesn't.

**Comment by cyrenajs** [comment]
It worked, @jakehilborn, thanks. Btw, yes, 2560x1440 is the second 'scaled' option in System Preferences, which is, it says it 'Looks like 2560x1440', although that's what I want to avoid mostly, in order to boost my mac's performance. I'm fine with unscaled 2560x1440. 

So displayplacer was able to lists my unscaled 2560x1440, and even marked it as current:

```
~$ displayplacer list | grep 2560x1440                                                                 
Resolution: 2560x1440
  mode 48: res:2560x1440 hz:60 color_depth:4 scaling:on
  mode 49: res:2560x1440 hz:60 color_depth:8 scaling:on
  mode 84: res:2560x1440 hz:30 color_depth:4 scaling:on
  mode 85: res:2560x1440 hz:30 color_depth:8 scaling:on
  mode 120: res:2560x1440 hz:50 color_depth:4 scaling:on
  mode 121: res:2560x1440 hz:50 color_depth:8 scaling:on
  mode 156: res:2560x1440 hz:24 color_depth:4 scaling:on
  mode 157: res:2560x1440 hz:24 color_depth:8 scaling:on
  mode 194: res:2560x1440 hz:25 color_depth:4 scaling:on
  mode 195: res:2560x1440 hz:25 color_depth:8 scaling:on
  mode 228: res:2560x1440 hz:60 color_depth:4 <-- current mode
  mode 229: res:2560x1440 hz:60 color_depth:8
  mode 258: res:2560x1440 hz:60 color_depth:4
  mode 259: res:2560x1440 hz:60 color_depth:8
  mode 310: res:2560x1440 hz:30 color_depth:4
  mode 311: res:2560x1440 hz:30 color_depth:8
  mode 360: res:2560x1440 hz:50 color_depth:4
  mode 361: res:2560x1440 hz:50 color_depth:8
  mode 412: res:2560x1440 hz:24 color_depth:4
  mode 413: res:2560x1440 hz:24 color_depth:8
  mode 466: res:2560x1440 hz:25 color_depth:4
  mode 467: res:2560x1440 hz:25 color_depth:8
```

And with using 'mode' it was able to set it correctly back and forth between the scaled and unscaled version.

**Comment by jakehilborn** [comment]
Great to hear!

**Comment by Ceylo** [comment]
Same issue here, currently doing VNC remote on a 5K iMac. I managed to lower to 4K, but I'm not able to turn off hidpi.

```
$ displayplacer list | grep 1920                                                                           
Resolution: 1920x1080
  mode 14: res:1920x1080 color_depth:4 scaling:on
  mode 15: res:1920x1080 color_depth:8 scaling:on <-- current mode
  mode 66: res:1920x1200 color_depth:4
  mode 67: res:1920x1200 color_depth:8
  mode 76: res:1920x1080 color_depth:4
  mode 77: res:1920x1080 color_depth:8
displayplacer "id:56EBFBC5-1F48-B8AB-49CC-1EFBD31C6CA6 res:1920x1080 color_depth:8 scaling:on origin:(0,0) degree:0"
```

I tried `displayplacer "id:56EBFBC5-1F48-B8AB-49CC-1EFBD31C6CA6 res:1920x1080 color_depth:8 scaling:off origin:(0,0) degree:0"` and `displayplacer "id:56EBFBC5-1F48-B8AB-49CC-1EFBD31C6CA6 res:1920x1080 color_depth:4 origin:(0,0) degree:0"` but they don't change anything 😕

**Comment by varenc** [comment]
#35 is the same issue.  The only way to get displayplacer to switch to an unscaled mode is by passing in the mode number.  It even turns scaling on for non-sense situations like when you're setting the resolution to your display's physical resolution.

In case it's helpful, I made a little script that takes in displayplacer settings, find the corresponding mode number, and then switches using that mode number: https://github.com/jakehilborn/displayplacer/issues/35#issuecomment-813142280

**Comment by jakehilborn** [comment]
@sarimarton @Ceylo @Ceylo The unscaled resolution issue has been fixed in [release v.1.30](https://github.com/jakehilborn/displayplacer/releases). Could you try it out and let me know if it works for you?

Thanks to @varenc for the bug fix PR.
